UnitTests/GitCommands.Tests/Settings/MockData/level1_repogit_GitExtensions.settings.xml

```diff
@@ -0,0 +1,27 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<dictionary>
+  <item>
+    <key>
+      <string>BuildServer.AppVeyor.AppVeyorProjectName</string>

```

Comment:
For these tests to have relevance, there should be tests that the effective data is correctly interpreted, i.e. with several build servers and overriding data.
To some extent, each build server internal setting dependency should be handled too...

Reply:
This test has these traits - there are 2 different build servers and 1 override.

The generic infra has no knowledge of specifics of each individual build server and hence can't test those here. And since there're plans to move the build server integration it'd be incorrect to add each build server specific tests here either.

Reply:
The override of BuildServerType should be checked
As the module provides functionality for settings for BuildServerTypes, it should test that.
Like `BuildServer.Type=MyServer` and `BuildServer.Type.MyServer=MySetting`, test overrides.

Reply:
> The override of BuildServerType should be checked
> As the module provides functionality for settings for BuildServerTypes, it should test that.
> Like `BuildServer.Type=MyServer` and `BuildServer.Type.MyServer=MySetting`, test overrides.

This is asserted in `RepoDistributed_settings_should_return_expected`.